src/java.base/share/classes/java/lang/foreign/AddressLayout.java

```diff
@@ -55,7 +54,6 @@
  * @see #ADDRESS_UNALIGNED
  * @since 19
```

Comment:
I seem to recall that `@since` should be adjusted to state the version in which we leave preview.

Reply:
Ok, I'll try to get confirmation on that, and then adjust the `@since` tags as needed.

Reply:
It seems that, at least this was also done in the case of Loom: https://github.com/openjdk/jdk/pull/13203/files#diff-2a5b4b651fa41e70f7953f7ee06e964d8396fcd1fe5755a1609e81b3f84d9224R842

Reply:
I've confirmed that we need to update the `@since` tags. This is also mentioned in [JEP12](https://bugs.openjdk.org/browse/JDK-8195734): 

> The author of a preview API element must also provide an `@since` tag in the element's javadoc comment, indicating the release when `@preview` was first added. If the API element is eventually made final and permanent in Java SE $N, then the `@since` tag must be changed to indicate release $N, as the element's history prior to $N is not of long-term interest.